docs/documentation/stories/universal-rendering.md

```diff
@@ -127,7 +138,7 @@ Then, remove the `"polyfills"` key - those aren't needed on the server, and adju
       "root": "src",
       // Build to dist-server instead of dist. This prevents
       // client and server builds from overwriting each other.
-      "outDir": "dist-server",
+      "outDir": "dist/dist-server",
```

Comment:
Bad idea to put the Universal files in the same directory as normal build, I've tried this option and it causes issues. Better stay in another directory.

Reply:
Thanks for the suggestion @cyrilletuzi I'll apply those changes now 😄  Just out of curiosity what are the issues caused by having it within the dist?

Reply:
I don't remember exactly, but good chances it was a problem of wrong import paths (as it's in a subdirectory) or a conflict on some building steps as files are in the same main directory.